packages/vx-tooltip/src/tooltips/TooltipWithBounds.tsx

```diff
@@ -1,44 +1,40 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { withBoundingRects } from '@vx/bounds';
```

Comment:
may need a `@ts-ignore` here as you mentioned because `@vx/bounds` currently has no types.